thirdparty/patches/AwsFindPackage.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
would you please point me the original location of this file, probably in aws repository source tree?

Reply:
https://github.com/awslabs/aws-c-common/blob/main/cmake/AwsFindPackage.cmake